From 3208c1114d6307a8f8a6ca2b000bfe306094bdf5 Mon Sep 17 00:00:00 2001
From: Wang Haoran <b50027@freescale.com>
Date: Mon, 10 Nov 2014 15:02:01 +0800
Subject: [PATCH 66/95] MA-5839-1 Fix compile error for upgrade Android-L

 The method is removed by AOSP. Use AOSP's release method due:
 MediaBuffer: ABuffer will release MediaBuffer when it's destructed.

Signed-off-by: Wang Haoran <b50027@freescale.com>
---
 media/libstagefright/wifi-display/source/Converter.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/media/libstagefright/wifi-display/source/Converter.cpp b/media/libstagefright/wifi-display/source/Converter.cpp
index 3d69bf6..7fbe493 100644
--- a/media/libstagefright/wifi-display/source/Converter.cpp
+++ b/media/libstagefright/wifi-display/source/Converter.cpp
@@ -1,6 +1,6 @@
 /*
  * Copyright 2012, The Android Open Source Project
- * Copyright (C) 2013 Freescale Semiconductor, Inc.
+ * Copyright (C) 2014 Freescale Semiconductor, Inc.
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
@@ -496,7 +496,7 @@ void Converter::refineVideoQueue()
             // if buffer repeat, delete it.
             mInputBufferQueue.erase(prev);
             // release buffer.
-            ReleaseMediaBufferReference(prevBuf);
+            prevBuf->setMediaBufferBase(NULL);
             bufSize --;
             if (bufSize <= requireSize) break;
         }
-- 
1.8.0

